### PR TITLE
Add option to remove winners from drawing and other improvements

### PR DESCRIPTION
**Pull request by andrewmh**
Other improvements include:
* keeping dark color when points go above the number of colors enumerated
* import tab separated data with points via function available in console
* fixing random logic for raffle with points to be more evenly distributed

### CODE REVIEW

File: js/raffle.js [hunk 1]
```diff
@@ -127,14 +140,38 @@ var removeDuplicateNames = function(data){
 	});
 }
 
+function standardizedImported() {
+	namePoints = {};
```
Comment (stringham) [inline js/raffle.js]: `var namePoints = {}`

Comment (andrewmh) [reply js/raffle.js]: The python in me coming out.

File: js/raffle.js [hunk 2]
```diff
@@ -77,6 +75,16 @@ function elementInViewport(el) {
   );
 }
 
+function remainingParticipants() {
+	participants = 0;
```
Comment (stringham) [inline js/raffle.js]: lol, here too.

Comment (andrewmh) [reply js/raffle.js]: I found a couple more. 😞 I think they're all fixed.

**Pull request merged**